Cargo.toml

```diff
@@ -33,7 +33,7 @@ casual = "0.2.0"
 clap_complete = "4.4.4"
 constcat = "0.4.0"
 curl = "0.4.44"
-fmutex = "0.1.0"
+fs2 = "0.4.3"
```

Comment:
I'm hesitant to use this crate because it is unmaintained, the last version was published 6 years ago! Looking on crates.io a good option to try might be [`fd-lock`](https://crates.io/fd-lock)